tests/components/modbus/conftest.py

```diff
@@ -39,9 +39,16 @@ def mock_pymodbus():
         yield mock_pb
 
 
-@pytest.fixture
-async def mock_modbus(hass, do_config):
+@pytest.fixture(
+    scope="function",
```

Comment:
`"function"` is the default scope I believe.

Reply:
You are right, removed!